### ISSUE TITLE
About the pullup values.

### ISSUE THREAD

**Issue by CelliesProjects** [stickbreaker/ESP32-i2c#1]
@stickbreaker Hi there.
Was browsing this repo when I saw these lines in your 'I2C_Scan.ino'.
````
/* I2C slave Address Scanner
for 5V bus
 * Connect a 4.7k resistor between SDA and Vcc
 * Connect a 4.7k resistor between SCL and Vcc
for 3.3V bus
 * Connect a 2.4k resistor between SDA and Vcc
 * Connect a 2.4k resistor between SCL and Vcc

 */
````
I use 4.7k resistors on my 3.3V i2c bus, as this is the 'usual' Arduino advice from the internet. ( I am no electrical engineer ) 
What is the reason for 2.4k resistors on 3.3V?

BTW does the one-wire lib you modified end up as pr for the paulstofregen or is it a real fork?

**Comment by stickbreaker** [comment]
@CelliesProjects I wrote my justification for pullup values at the tail end of [issue #15](https://github.com/stickbreaker/arduino-esp32/issues/15)

edit: created a Wiki for this question [Pullup Calculations](https://github.com/stickbreaker/arduino-esp32/wiki/Pullup-Values)

the OneWire is just for a pr.  Paul has agreed to merge it with the main branch.  He wanted a couple of thematic changes before he would accept it, I have made the changes. I am just waiting from him to review it.  After he merges it. I will delete my fork.  

This arduino-esp32 fork is also temporary.  Though, it will have a longer duration. @me-no-dev is reviewing it and changing the style.  He will probable gut it, transmogrify it without actually 'merging'.
I am using this fork to work on SLAVE mode. After I get SLAVE mode functioning and 'merged' into the main branch this fork will be pruned.
Chuck.
